packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -390,41 +400,76 @@ const Autocomplete = <
       (item: ReactElement & { children?: ReactElement[] }) =>
         [item].concat(item.children || []),
     );
-
-    // the height of an Odyssey autocomplete option item that is used to calculate height of window
-    const optionHeight = 45; //px
+    const sizeMap = useMemo(() => new Map<number, number>(), []);
 
     // The number of items (rows or columns) to render outside of the visible area for performance and scrolling reasons
     const overscanRowCount = 8;
 
-    const itemSize = useCallback(() => optionHeight, []);
-
     const gridRef = useResetCache(itemData.length);
 
+    const setItemSize = useCallback<SetItemSize>(
+      (index, size) => {
+        gridRef?.current?.resetAfterIndex(0, true);
+        sizeMap.set(index, size);
+      },
+      [gridRef, sizeMap],
+    );
+
+    const getItemSize = useCallback(
+      (index: number) => sizeMap.get(index) || 45,
+      [sizeMap],
+    );
```

Comment:
Can we move this magic number as a const at the top?

Or just use `0`. If it breaks styling, which it won't, that's fine.